openedx/features/course_experience/static/course_experience/js/CoursetalkReviews.js

```diff
@@ -0,0 +1,28 @@
+/*
+  Enable users to switch between viewing and writing CourseTalk reviews.
+ */
+
+export class CourseTalkReviews {  // eslint-disable-line import/prefer-default-export
+  constructor() {
+    const $courseTalkToggleReadWriteReviews = $('.toggle-read-write-reviews');
+    const $courseTalkScriptObject = $('.coursetalk-reviews-script');
+
+    const readSrc = '//d3q6qq2zt8nhwv.cloudfront.net/s/js/widgets/coursetalk-read-reviews.js';
+    const writeSrc = '//d3q6qq2zt8nhwv.cloudfront.net/s/js/widgets/coursetalk-write-reviews.js';
+
+    $coursetalkToggleReadWriteReviews.on('click', () => {
+      // Cache js file for future button clicks
+      $.ajaxSetup({ cache: true });
+
+      // Toggle the new coursetalk script object
+      const showReadView = $courseTalkScriptObject.attr('src') == writeSrc;
+      const newSource = showReadView ? readSrc : writeSrc;
+      $courseTalkScriptObject.attr('src', newSource);
+      $.getScript(newSource);
+
+      // Button text to toggle user to the other view
+      const newText = showReadView ? 'Write a Review' : 'View Reviews';
```

Comment:
This is supposed to be on the staging environment as of today, is it a global function or do we need to import it explicitly? @andy-armstrong